android-client-sdk/src/test/java/com/devcycle/sdk/android/api/DevCycleClientTests.kt

```diff
@@ -359,7 +359,9 @@ class DevCycleClientTests {
             }
         }
         client.identifyUser(DevCycleUser.builder().withUserId("new_userid").build())
+        Thread.sleep(100) // Add small delay
```

Comment:
adding this delay allows the test to not flake, they can be removed in the future